src/serac/physics/integrators/traction_integrator.cpp

```diff
@@ -0,0 +1,226 @@
+// Copyright (c) 2019-2021, Lawrence Livermore National Security, LLC and
+// other Serac Project Developers. See the top-level LICENSE file for
+// details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+
+#include "serac/physics/integrators/traction_integrator.hpp"
+
+namespace serac::mfem_ext {
+
+void TractionIntegrator::AssembleFaceVector(const mfem::FiniteElement&        element_1, const mfem::FiniteElement&,
```

Comment:
Another way to do this would be "AssembleFaceResidual" and "AssembleFaceTangent", which might be better going forward

Reply:
This is another MFEM-ism that we unfortunately inherited through the NonlinearForm/NonlinearIntegrator class.